tests/test_security.c

```
@@ -1,402 +1,293 @@
\ No newline at end of file
+}
+
+int main(void) {
+    if (sodium_init() < 0) {
+        fprintf(stderr, "libsodium init failed\n");
+        return 1;
+    }
+
+    int failures = 0;
+
+    if (test_init()) failures++;
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)